openapi-mock/server/app.js

```diff
@@ -66,13 +66,6 @@ function configValidation(configJson) {
       if (!api.baseurl || !api.baseurl.match(matchRegex)) {
         error_message += "\napi " + api.name + ": baseurl does not exist or is in the wrong format";
       }
-      if (!api.metadata || !api.metadata.match(matchRegex)) {
```

Comment:
why the regexp check is removed?

Reply:
Done....Now checking for the regex when the endpoint exists